### ISSUE TITLE
Fonts in code blocks are not rendered correctly

### ISSUE THREAD

**Issue by lfrestrepog** [plfa/plfa.github.io#582]
This is probably related to issue #579 

The fonts in code block no longer render correctly in several web browsers. It seems the computed font-family in the final css has an incorrect font. Tested with firefox and microsoft edge.

Here's what I believe to be the relevant parts of the css:
```css
pre.Agda {
    font-family: 'DejaVu mononoki Symbola Droid','mononoki','DejaVu Sans Mono','Source Code Pro','Bitstream Vera Sans Mono','FreeMono','Courier New','Monaco','Menlo',monospace,serif;
    font-size: .85em;
}
```

and

```css
code {
    font-family: 'DejaVu mononoki Symbola Droid','mononoki','DejaVu Sans Mono','Source Code Pro','Bitstream Vera Sans Mono','FreeMono','Courier New','Monaco','Menlo',monospace,serif;
    font-size: .85em;
    white-space: pre;
}
```

I'm not familiar with the site generation tooling in use, but it should be something easy to fix. Maybe somebody could point me in the right direction?

Thanks a lot.

**Comment by wenkokke** [comment]
I'm guessing this has something to do with the "DejaVu mononoki Symbola Droid" font? I don't know enough about fonts to figure out what's wrong with it, though... @Altariarite, do you know what's going on?

**Comment by wenkokke** [comment]
Closed as a duplicate of #591, as that one has a screen shot. @lfrestrepog, could you confirm if that's the output you're seeing as well?

**Comment by lfrestrepog** [comment]
Yes, that's correct.
